Menu.php

```diff
@@ -91,7 +91,8 @@ public function configureTagManagerMenu(MenuTagManager $menu)
             if (!empty($container)) {
 
                 $params = array('idContainer' => $idContainer); // not needed as it is already present in url but we make sure the id is set
-                $menuCategory = $container['name'];
+                $menuCategory = strlen($container['name']) > 50 ? substr($container['name'], 0, 50) . '...' : $container['name'];
```

Comment:
Better to use an ellipsis here instead of 3 dots
```suggestion
                $menuCategory = strlen($container['name']) > 50 ? substr($container['name'], 0, 50) . '…' : $container['name'];
```

Reply:
Thank you :+1: